lib/hooks/ReturnStatement.js

```diff
@@ -42,7 +42,7 @@ exports.getIndentEdges = function(node, opts) {
   return parentheses ?
     {
       startToken: parentheses.opening,
-      endToken: parentheses.closing
+      endToken: parentheses.closing.prev
```

Comment:
uhm.. I guess I could update the logic a little bit and avoid all these changes.